python_dependencies/Pipfile

```diff
@@ -13,7 +13,7 @@ requests = "==2.22.0"
 # handle service dependencies
 cachetools = "*"
 configparser = "*"
-pymongo = "*"
+pymongo = "==3.11.0"
```

Comment:
Originally pipenv auto upgraded to v4.X which cause the handle service tests to start failing, so I pinned it and the test magically started working again. 
@Tianhao-Gu heads up